sympy/matrices/matrices.py

```diff
@@ -2377,10 +2377,10 @@ def jordan_cell_power(jc, n):
                     bn = binomial(n, i)
                     if isinstance(bn, binomial):
                         bn = bn._eval_expand_func()
-                    jc[0, i] = l**(n-i)*bn
+                    jc[0,i] = l**(n-i)*bn
```

Comment:
I'm going to merge this anyway but just to be clear the spacing above was fine. I wasn't suggesting to change that.

PEP 8 would suggest
```
jc[0, i] = l ** (n - i) * bn
```
I don't think that that spacing on the rhs really helps. Probably SymPy is something of a special case as it deals with a lot of mathematical expressions so I would go with
```
jc[0, i] = l**(n-i)*bn
```
as was there before.

Reply:
Just to let you know I am 0% political on formatting, I try to match what I am working on and if I do not it is because of `force_of_habit()**unnoticed`. Personally I prefer lots of spaces and double spaced lines for maximum readability but with context coloring it doesn't really matter.